### PR TITLE
Add Web messenger webview support entry to release notes

### PR DESCRIPTION
**Pull request by lemieux**
Pretty much what the title says.

### CODE REVIEW

File: pages/changelog.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ layout: two-column
 
 # Changelog
 
+## October 23rd, 2017
+- Added support for [Conversation Extensions](/guide/conversation-extensions/)!
+  - Web Messenger [4.1.0](https://github.com/smooch/smooch-web/releases/tag/4.1.0) released with support for extensions
+
```
Comment (jpjoyal) [inline pages/changelog.md]: I would add A note on the react update too

File: pages/changelog.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ layout: two-column
 
 # Changelog
 
+## October 23rd, 2017
+- Added support for [Conversation Extensions](/guide/conversation-extensions/)!
```
Comment (jpjoyal) [inline pages/changelog.md]: I would remove this line

File: pages/changelog.md (hunk 1 above)
Comment (lemieux) [reply pages/changelog.md]: Really? seems like a low level detail now that people don't need to install React as a dependency anymore. There's a note in the Web Messenger release notes. I don't mind, just curious about the level of details in these notes

File: pages/changelog.md (hunk 2 above)
Comment (lemieux) [reply pages/changelog.md]: :+1: wasn't sure if I should include it or not

File: pages/changelog.md (hunk 1 above)
Comment (jugarrit) [reply pages/changelog.md]: I would tend to think this would be only in the release notes on Github but I suppose it couldn't hurt

Comment (lemieux) [reply pages/changelog.md]: there you go

Comment (jpjoyal) [reply pages/changelog.md]: I liked the performance increase mention in the release and thought it was useful here too

Comment (lemieux) [reply pages/changelog.md]: yeah good idea, I'll add the perf and speed thing

Comment (lemieux) [reply pages/changelog.md]: updated

**Pull request merged**